m03_acos/create_acos_exercises.rst

```diff
@@ -0,0 +1,454 @@
+          "content":"<img src=\"/static/draganddrop-data-management/exp_folder.png\" alt=\"folder\">",
+          "reuse":false,
+          "feedback":{
+            "folder":"Correct!, this folder contain the results of different experiments",
+            "DEFAULT":"This folder contains the results of different experiments."
+          }
+        },
+        "file":{
+          "content":"<img src=\"/static/draganddrop-data-management/file.png\" alt=\"folder\">",
+          "reuse":true,
+          "feedback":{
+            "folder":"Correct!, this is a group of files",
+            "DEFAULT":"This is a list of different files."
+          }
+        },
+        "readme":{
+          "content":"<img src=\"/static/draganddrop-data-management/readme.png\" alt=\"folder\">",
+          "reuse":true,
+          "feedback":{
+            "folder":"Correct!, This is a readme file",
+            "DEFAULT":"This is a readme file that contains meta data.."
+          }
+        }
+      },
+      "droppables":{
+        "source":{
+          "correct":[
+            "source"
+          ]
+        },
+        "date_folders":{
+          "correct":[
+            "date_folder"
+          ]
+        },
+        "experiments":{
+          "correct":[
+            "experiment"
+          ]
+        },
+        "files":{
+          "correct":[
+            "file"
+          ]
+        },
+        "readme":{
+          "correct":[
+            "readme"
+          ]
+        }
+      }
+    }
+  
+Once your Acos exercise is ready, and the exercise files have been created, you will have two choices for testing your code locally in the browser. :ref:`The first option <local-test>` is to test the exercise with a copy of the Acos-server in your computer. :ref:`The second option <docker-test>` is to test the exercise by using a pre-installed docker container which is downloaded after you have compiled an A+ course for the first time. 
+
+.. warning:: Before proceeding to test your Acos exercise, you should :ref:`configure the localhost <acos-docker-mac-windows>` in your computer. Therefore, you should verify that your host file have the ``127.0.0.1 acos`` configuration.
+
+
+.. _local-test:
+Test Acos activities locally
+------------------------------
+In order to test Acos exercises locally, you should download the `acos-server repository <https://github.com/acos-server/acos-server>`_ from GitHub. Once you have downloaded this file, you should link the :ref:`Acos exercise you have developed <create-acos-exercise>` to the local Acos-server.
+
+1. Open the terminal in the exercise project directory that you created :ref:`above <create-acos-exercise>`, and type:
+
+.. code-block:: bash
+
+  sudo npm link
+
+2. Open a terminal in the Acos server project directory and type: 
+  
+.. code-block:: bash
+
+  sudo npm link acos-draganddrop-data-management
+
+(the ``acos-draganddrop-data-management`` name will vary according to the name of your package).
```

Comment:
Would it be better if "The" was capitalized and the period inside the parentheses?